native/cocos/platform/interfaces/modules/Device.cpp

```diff
@@ -31,9 +31,28 @@
 #include "platform/interfaces/modules/INetwork.h"
 #include "platform/interfaces/modules/IScreen.h"
 #include "platform/interfaces/modules/IVibrator.h"
+#include "platform/interfaces/modules/ISystemWindow.h"
 
 namespace cc {
 
+int Device::getInnerWidth() {
+    CCASSERT(CC_GET_PLATFORM_INTERFACE(ISystemWindow) != nullptr, "Screen interface does not exist");
+    auto viewSize = CC_GET_PLATFORM_INTERFACE(ISystemWindow)->getViewSize();
+    return static_cast<int>(viewSize.x);
+}
+
+int Device::getInnerHeight() {
+    CCASSERT(CC_GET_PLATFORM_INTERFACE(ISystemWindow) != nullptr, "Screen interface does not exist");
+    auto viewSize = CC_GET_PLATFORM_INTERFACE(ISystemWindow)->getViewSize();
+    return static_cast<int>(viewSize.y);
+}
+
+uint32_t Device::getWindowHandler() {
+    CCASSERT(CC_GET_PLATFORM_INTERFACE(ISystemWindow) != nullptr, "Screen interface does not exist");
```

Comment:
The same as above.

Reply:
this is 3.5, not replace with CC_ASSERT.

Reply:
It is new codes, please use `CC_ASSERT` as `CCASSERT` is removed in v3.6. And these codes will be merged to develop in future.